### ISSUE TITLE
Open all external links with another browser

### ISSUE THREAD

**Issue by kaffeeringe** [darktrojan/openwith#110]
I would like to have Facebook, Twitter, and so on in one browser and all other pages in another browser. Your plugin is already very helpful. But it would be cool if I could open all links in the other browser. 

So, when I am on Facebook all links that are not also facebook.com should open in the other browser.

**Comment by Mentis** [comment]
Second this! I would like to create separate profiles for some web applications and open all external links in default firefox profile. 
It would be very helpful!

**Comment by singpolyma** [comment]
Yeah, the ability to pass URLs that match a certain pattern to a different application automatically would be a huge win!

**Comment by sam-roth** [comment]
> Yeah, the ability to pass URLs that match a certain pattern to a different application automatically would be a huge win!

@singpolyma's request would be really useful for the G Suite version of Hangouts, which doesn't support Firefox (unlike the free version, which supposedly does). Even Google Meet, which is supposed to work with Firefox, doesn't actually work with Firefox.

**Comment by sam-roth** [comment]
@singpolyma It's not listed under [Web browsers that work with Hangouts](https://support.google.com/hangouts/answer/2944865) and I get an error if I try to join a hangout.

**Comment by 7MXek** [comment]
This would be a huge help for a little side project of mine using this extension.
